charts-community-modules/ag-charts-community/src/chart/legend.ts

```diff
@@ -808,29 +808,42 @@ export class Legend {
                 (ls, s) => [...ls, ...s.getLegendData()],
                 [] as Array<LegendDatum>
             );
+
             const visibleItemsCount = legendData.filter((d) => d.enabled).length;
             const clickedItem = legendData.find((d) => d.itemId === itemId && d.seriesId === seriesId);
+
+            const seriesItemCounts = legendData.reduce((acc, d) => {
+                acc[d.seriesId] ? ++acc[d.seriesId] : (acc[d.seriesId] = 1);
```

Comment:
Nit: There's quite a bit going on here, could we break this down a bit to make it more readable, e.g.:
```suggestion
                acc[d.seriesId] ??= 0;
                acc[d.seriesId]++;
```